aries-site/src/examples/components/table/TableSingleSelectExample.js

```diff
@@ -1,5 +1,23 @@
 import React from 'react';
-import { Box, DataTable, Heading, Text, ResponsiveContext } from 'grommet';
+import {
+  Box,
+  DataTable,
+  Heading,
+  Text,
+  ResponsiveContext,
+  Button,
+} from 'grommet';
+import { FormPrevious } from 'grommet-icons';
+
+const orderDetails = [
+  'Order Name:',
```

Comment:
also it seems like the designs don't use a ":" so I think we can remove that in all of these and just have the text.

Reply:
your right I think I added that by habbit